homeassistant/components/jewish_calendar/strings.json

```
@@ -33,5 +33,141 @@
         }
       }
     }
+  },
+  "services": {
+    "get_gregorian_date": {
+      "name": "Get Gregorian Date",
+      "description": "Returns Hebrew Calendar information for a given Gregorian date.  Pass one or more include_* parameters to specify which fields to return.",
+      "fields": {
+        "date": {
+          "name": "Date",
+          "description": "Look up the specified date or time.  If you pass a date with a time, it will be evaluated relative to sunset, returning the following day if after sunset (not candle lighting). Defaults to today (relative to midnight, not sunset)."
+        },
+        "include_hebrew_date_info": {
+          "name": "Include Hebrew Date Info",
+          "description": "True to return Hebrew year, month, day, and related calendar information."
+        },
+        "include_holiday_info": {
+          "name": "Include Holiday Info",
+          "description": "True to return information about the holiday that falls on this date, if any."
+        },
+        "include_zmanim": {
+          "name": "Include Zmanim",
+          "description": "True to return the day's zmanim timestamps."
+        }
+      }
+    },
+    "get_gregorian_date_range": {
+      "name": "Get Gregorian Date Range",
+      "description": "Returns Hebrew Calendar information for a range of Gregorian dates.  Pass one or more include_* parameters to specify which fields to return.",
+      "fields": {
+        "date": {
+          "name": "Date",
+          "description": "The first day to return.  If you pass a date with a time, it will be evaluated relative to sunset, returning the following day if after sunset (not candle lighting). Defaults to today (relative to midnight, not sunset)."
+        },
+        "number_of_days": {
+          "name": "Number of Days",
+          "description": "The number of days to return, including 'date'."
+        },
+        "include_hebrew_date_info": {
+          "name": "Include Hebrew Date Info",
```

Don't repeat the translations, refer back instead, or if PR 118783 is accepted move these to a common section https://github.com/home-assistant/core/pull/118783:
```suggestion
          "name": "[%key:componentn::jewish_calendar::...include_hebrew_date_info::name%]",
```

---

Is there any documentation for exactly how these keys work?

---

I don't think it's translated, however simply the full key name which you're referring to. hassfest ensures the reference is valid.
I updated my example to be complete and hopefully less confusing.

---

PR https://github.com/home-assistant/core/pull/118783, it'd probably be cleaner to add the shared strings to a `common`-section

---

Done